packages/store/src/Utils.sol

```diff
@@ -6,6 +6,10 @@ library Utils {
     return a / b + (a % b == 0 ? 0 : 1);
   }
 
+  function toString(uint256 input) internal pure returns (string memory) {
+    return string(abi.encodePacked(input));
+  }
```

Comment:
as a general utility this is a bit misleading, as it's specifically meant for table ids

Reply:
would echo that above! I personally use `toString(uint)` to mean "convert uint to a string of digits" rather than "convert this uint binary to a string binary"

Reply:
Maybe we combine this with `ResourceSelector`, move it out into a more general purpose thing (i.e. `common` package in the future)?

Reply:
happy to change it to `tableIdToString(uint256)`. I'm a bit hesitant to use `ResourceSelector` here because that's a _World_ concept and i'd like to allow _Store_ to be used with other types of ids that are not ResourceSelectors (maybe someone wants to use a `keccak256` id, or maybe a different type of selector that's not just namespace and file but other fragments)

If the `tableId` is not representable as string, the error log just includes some garbage characters, but I don't think that hurts.